var/spack/repos/builtin/packages/root/package.py

```diff
@@ -452,6 +452,8 @@ class Root(CMakePackage):
         "cxxstd=20", when="@:6.28.02", msg="C++20 support requires root version at least 6.28.04"
     )
 
+    conflicts("%gcc@10:", when="cxxstd=20")
```

Comment:
This can't be right. No gcc newer than gcc-10 can handle C++20?!?

Reply:
Does this need to apply to just gcc-10?
```suggestion
    conflicts("%gcc@10", when="cxxstd=20")
```

Reply:
I just messed up the range.

I guess it should be

```suggestion
    conflicts("%gcc@:10", when="cxxstd=20")
```

since I don't think lower versions would realistically work either?